pkg/services/cloudmigration/api/api.go

```diff
@@ -47,10 +48,46 @@ func (cma *CloudMigrationAPI) registerEndpoints() {
 		cloudMigrationRoute.Post("/migration/:uid/run", routing.Wrap(cma.RunMigration))
 		cloudMigrationRoute.Get("/migration/:uid/run", routing.Wrap(cma.GetMigrationRunList))
 		cloudMigrationRoute.Get("/migration/run/:runUID", routing.Wrap(cma.GetMigrationRun))
+		cloudMigrationRoute.Get("/token", routing.Wrap(cma.GetToken))
 		cloudMigrationRoute.Post("/token", routing.Wrap(cma.CreateToken))
 	}, middleware.ReqOrgAdmin)
 }
 
+// swagger:route GET /cloudmigration/token migrations getCloudMigrationGetToken
+//
+// Fetch the cloud migration token if it exists.
+//
+// Responses:
+// 200: cloudMigrationGetTokenResponse
+// 401: unauthorisedError
+// 404: notFoundError
+// 403: forbiddenError
+// 500: internalServerError
+func (cma *CloudMigrationAPI) GetToken(c *contextmodel.ReqContext) response.Response {
+	ctx, span := cma.tracer.Start(c.Req.Context(), "MigrationAPI.GetToken")
+	defer span.End()
+
+	logger := cma.log.FromContext(ctx)
+
+	token, err := cma.cloudMigrationService.GetToken(ctx)
+	if err != nil {
+		if !errors.Is(err, cloudmigration.ErrTokenNotFound) {
+			logger.Error("fetching cloud migration access token", "err", err.Error())
```

Comment:
Since `ErrTokenNotFound` is created with `errutil.NotFound` the http response created by `return response.ErrOrFallback(http.StatusInternalServerError, "fetching cloud migration access token", err)` would have the 404 status, right?

Reply:
ah right, cool